drivers/gles3/storage/config.cpp

```diff
@@ -77,7 +77,7 @@ Config::Config() {
 	}
 #endif
 
-	bptc_supported = extensions.has("GL_ARB_texture_compression_bptc") || extensions.has("EXT_texture_compression_bptc");
+	bptc_supported = extensions.has("GL_ARB_texture_compression_bptc") || extensions.has("GL_EXT_texture_compression_bptc") || extensions.has("EXT_texture_compression_bptc");
```

Comment:
I think Emscripten reports all the extensions with the `GL_` prepended (as well as the version without), so if `"EXT_texture_compression_bptc"` is for WebGL, I don't think it's necessary, it should be covered by the `GL_` version too. I haven't tested this, though, this is just from memory of working on other WebGL issues in the past

Reply:
Or, actually, we do that - not Emscripten. If you look about 20 lines above this code, you'll see that we're adding all the extensions to the list twice, once without the `GL_` and once with.

So, for sure, we don't need both here:

```suggestion
	bptc_supported = extensions.has("GL_ARB_texture_compression_bptc") || extensions.has("GL_EXT_texture_compression_bptc");
```

Reply:
Alright, I removed the checks without the "GL_" prefix. But this makes me realize that Godot is probably adding "GL_" in front of the "W

Reply:
Yep, just noticed that too while checking haha. I should have checked back here first